scylladb/sphinx-scylladb-theme
# Add enlarge images button

Closes https://github.com/scylladb/sphinx-scylladb-theme/issues/187

## How to test this PR

1. Clone this PR. For more information, see [Cloning pull requests locally](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally).

2. Enter the docs folder, and run:

```
make preview
````

3. Open http://127.0.0.1:5500/ with your favorite browser. Then, navigate to "Examples" > "Images". At the end of the page, you will find an example on how to use the custom class "enlarge-image".

## Screenshots

![image](https://user-images.githubusercontent.com/9107969/142157724-c9391186-832e-49c5-9c86-efd9fe9fa235.png)

![image](https://user-images.githubusercontent.com/9107969/142158238-bd2896f4-353d-455c-9f26-548442eb8c72.png)
